.eslintrc.js

```diff
@@ -282,9 +287,6 @@ module.exports = {
     },
   ],
   settings: {
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts', '.tsx'],
```

Comment:
This is included already with the import/typescript extends: https://github.com/import-js/eslint-plugin-import/blob/main/config/typescript.js